### PR TITLE
comments

### PR DESCRIPTION
**Pull request by angusjf**
image

### CODE REVIEW

File: src/components/RegisterStack.tsx [hunk 1]
```diff
@@ -114,7 +114,7 @@ export const EmailForm: FC<EmailFormProps> = ({ navigation }) => {
         value={email || ""}
         onChangeText={setEmail}
         validity={validity}
-        long
+        length={7}
```
Comment (danigo99) [inline src/components/RegisterStack.tsx:117]: does length add some validation checks here? If you have a long name are you screwed lol?

File: src/firebaseClient.ts [hunk 2]
```diff
@@ -55,6 +56,24 @@ export const updateSignature = async (user: User, signature: string) => {
   set(ref(db, `users/${user.userId}/signature`), signature);
 };
 
+export const addComment = async (
+  commentee: User,
+  timestamp: number,
+  comment: Comment
+) => {
+  const db = getDatabase(firebaseApp);
+  console.log(comment);
```
Comment (danigo99) [inline src/firebaseClient.ts]: can we remove?

File: src/firebaseClient.ts [hunk 3]
```diff
@@ -55,6 +56,24 @@ export const updateSignature = async (user: User, signature: string) => {
...
+  set(
```
Comment (danigo99) [inline src/firebaseClient.ts:65]: await?

File: src/components/RegisterStack.tsx (hunk 1 above)
Comment (angusjf) [reply src/components/RegisterStack.tsx:117]: not sure lol

**Pull request merged**